Source/Application.h

```diff
@@ -49,19 +53,19 @@ class Application
 	ModuleProgram* program;
 	ModuleResources* resources;
 	ModuleCamera* camera;
-#ifndef ENGINE
 	ModulePlayer* player;
-#endif // ENGINE
 	ModuleDebugDraw* debug;
 	ModuleEditor* editor;
 
 private:
 	std::vector<std::unique_ptr<Module> > modules;
 	std::unique_ptr<Timer> appTimer;
+	std::unique_ptr<Timer> onPlayTimer;
```

Comment:
Now that I whink about it, maybe both timers could be references? Maybe it's not possible.
```suggestion
	Timer onPlayTimer;
```

Reply:
I don't think so. In this case both timers are only used in the application. And they are used during the whole execution, so I think it is worth to have them unique to not have to deal with the delete.

Reply:
Sorry, that's not what I mean. I think you are thinking about using a `Timer*`. In which case, yeah, I think it's better to have it as a unique pointer. But I'm wondering if it could be a plain `Timer`, since it will never be null and it doesn't need to be passed around. If I'm not mistaken it should also be more efficient, but I could be wrong. As I said in the original comment, maybe it's not possible, but wanted to ask just in case.

Reply:
Ahhh, a see your point. I Will change thx.